### PR TITLE
added .htaccess files

### PR DESCRIPTION
**Pull request by chemix**
for critical folders app, temp, log

because someone could forget to add these files, if he move document root and index.php to root folder

### CODE REVIEW

File: temp/.htaccess [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Order Allow,Deny
+Deny from all
```
Comment (JanTvrdik) [inline temp/.htaccess]: missing new line at the end of file

File: log/.htaccess [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Order Allow,Deny
+Deny from all
```
Comment (JanTvrdik) [inline log/.htaccess]: missing new line at the end of file

**Comment by JanTvrdik** [comment]
My guess is that they are missing intentionally.

**Comment by chemix** [comment]
@matej21 use this project at workshop tommorow, and someone ask what happens on traditional czech hosting, where document root is root of repository. I think, it's proactive secure thing

**Comment by VojtaStanek** [comment]
I think that there is no reason for `.htaccess` files to not be there.
When you are using a "traditional czech hosting" it would be a big plus. Maybe there can be also `.htaccess` in root directory where will be rewriting to the file for "traditional czech hosting". That's one of the things that anyone deploying there (I think that's a majority of people). 
When you are using other hosting where you can set `www` to be root, it doesn't matter if there are these `.htaccess`s or not.

**Pull request closed without merging**